ordered_logistic_lpmf.hpp is messed up in C++11

#### Summary:

It has `>>&` on line 197
https://github.com/stan-dev/math/blob/develop/stan/math/prim/mat/prob/ordered_logistic_lpmf.hpp#L197
which yields from clang++
```
ordered_logistic_lpmf.hpp:197:73: warning: use of right-shift operator ('>>') in template argument will require
      parentheses in C++11 [-Wc++11-compat]
                                                T_cut, Eigen::Dynamic, 1>>& c) {
```

#### Description:

This leads to a compiler error.


#### Reproducible Steps:

Compile something with `ordered_logistic_lpmf`

#### Current Output:

Above compiler error

#### Expected Output:

No compiler error

#### Additional Information:


#### Current Version:
v2.17.0

--- a/stan/math/prim/mat/meta/broadcast_array.hpp
+++ b/stan/math/prim/mat/meta/broadcast_array.hpp
@@ -9,7 +9,7 @@ namespace stan {
   namespace math {
     namespace internal {
       template <typename ViewElt, typename OpElt, int R, int C>
-      class empty_broadcast_array<ViewElt, Eigen::Matrix<OpElt, R, C>> {
+      class empty_broadcast_array<ViewElt, Eigen::Matrix<OpElt, R, C> > {
       public:
         empty_broadcast_array() {}
        /**

--- a/stan/math/prim/mat/prob/ordered_logistic_lpmf.hpp
+++ b/stan/math/prim/mat/prob/ordered_logistic_lpmf.hpp
@@ -194,7 +194,7 @@ namespace stan {
     ordered_logistic_lpmf(const std::vector<int>& y,
                           const Eigen::Matrix<T_loc, Eigen::Dynamic, 1>& lambda,
                           const std::vector<Eigen::Matrix<
-                                                T_cut, Eigen::Dynamic, 1>>& c) {
+                                            T_cut, Eigen::Dynamic, 1> >& c) {
       typename return_type<T_loc, T_cut>::type logp_n(0.0);
 
       using std::exp;
@@ -237,7 +237,7 @@ namespace stan {
     ordered_logistic_lpmf(const std::vector<int>& y,
                           const Eigen::Matrix<T_loc, Eigen::Dynamic, 1>& lambda,
                           const std::vector<Eigen::Matrix<
-                                                T_cut, Eigen::Dynamic, 1>>& c) {
+                                            T_cut, Eigen::Dynamic, 1> >& c) {
       return ordered_logistic_lpmf<false>(y, lambda, c);
     }
   }

--- a/stan/math/prim/mat/prob/ordered_probit_lpmf.hpp
+++ b/stan/math/prim/mat/prob/ordered_probit_lpmf.hpp
@@ -171,7 +171,7 @@ namespace stan {
     ordered_probit_lpmf(const std::vector<int>& y,
                         const Eigen::Matrix<T_loc, Eigen::Dynamic, 1>& lambda,
                         const std::vector<Eigen::Matrix<
-                                              T_cut, Eigen::Dynamic, 1>>& c) {
+                                          T_cut, Eigen::Dynamic, 1> >& c) {
       using std::exp;
       using std::log;
 
@@ -214,7 +214,7 @@ namespace stan {
     ordered_probit_lpmf(const std::vector<int>& y,
                         const Eigen::Matrix<T_loc, Eigen::Dynamic, 1>& lambda,
                         const std::vector<Eigen::Matrix<
-                                              T_cut, Eigen::Dynamic, 1>>& c) {
+                                          T_cut, Eigen::Dynamic, 1> >& c) {
       return ordered_probit_lpmf<false>(y, lambda, c);
     }
   }

